<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Easy ocr</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="labelTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Select stream</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButtonPrev">
      <property name="text">
       <string>Prev</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="pushButtonNext">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLicense">
    <property name="title">
     <string>License</string>
    </property>
    <addaction name="actionQt"/>
    <addaction name="actionReadMe"/>
    <addaction name="actionContactMe"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSaveSettings"/>
    <addaction name="actionLoadSettings"/>
   </widget>
   <addaction name="menuLicense"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQt">
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionReadMe">
   <property name="text">
    <string>Read me</string>
   </property>
  </action>
  <action name="actionContactMe">
   <property name="text">
    <string>Contact me</string>
   </property>
  </action>
  <action name="actionSaveSettings">
   <property name="text">
    <string>Save settings</string>
   </property>
  </action>
  <action name="actionLoadSettings">
   <property name="text">
    <string>Load settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
